pyproject.toml

```diff
@@ -18,9 +18,6 @@ maintainers = [
 ]
 classifiers = [
     "Development Status :: 5 - Production/Stable",
-    "Framework :: Django :: 3.2",
-    "Framework :: Django :: 4.0",
-    "Framework :: Django :: 4.1",
     "Framework :: Django :: 4.2",
     "Framework :: Django :: 5.0",
     "Framework :: Django :: 5.1",
```

Comment:
+ Add Django 5.2

Reply:
Thanks for suggesting but all classifiers were already present on `master` from 430671cc2d55f148029985c6c4de11ce6bd9feb1. They just don't show on GitHub for this PR unless you click the “expnad diff” buttons:

<img width="1102" alt="Xnapper-2025-03-06-23 40 51" src="https://github.com/user-attachments/assets/aecc9599-05ad-4977-af4f-f0e2e71693f1" />

Reply:
Thank you, sorry I didn't notice. My apologies.